exercise/templatetags/exercise.py

```diff
@@ -161,24 +162,24 @@ def get_points_class(context):
         'not_passed': 'badge-danger',
         'missing': 'badge-danger',
     }
-    class_name = options.get(next(filter(lambda x: x != "unconfirmed", points_type), ""), "")
-    if "unconfirmed" in points_type:
+    class_name = options.get(points_type[0], "")
+    if points_type[1]:
         class_name += " unconfirmed-points"
     return class_name
 
 # Adds the content of the points_badge.
 @register.simple_tag(takes_context=True)
 def get_points_text(context):
     points_string = str(context['points']) + " / " + str(context['max'])
-    points_type = next(filter(lambda x: x != "unconfirmed", _get_points_type(context)), "")
+    points_type = _get_points_type(context)
```

Comment:
You can assign to two variables here. Makes code more readable ;)

Reply:
I made the `_get_points_class` function use that two variable setting. `get_points_text` only needs the first element of the tuple so I did not create a useless variable for the `unconfirmed` status.